src/SSOTokenGenerator.php

```diff
@@ -36,33 +36,19 @@ class SSOTokenGenerator
      *
      * @return string Encoded token.
      */
-    public static function createSignedTokenFromData(string $privateKey, array $tokenData, Signer $signer = null): string
+    public static function createSignedTokenFromData(string $privateKey, array $tokenData, Signer $signer = null, array $headers = []): string
```

Comment:
Where is `headers` used?

Reply:
It was an intermediate step. I removed it.